src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BranchScanningTest.java

```diff
@@ -66,13 +66,13 @@
     @Test
     public void uriResolverTest() throws Exception {
         BitbucketSCMSource source = getBitbucketSCMSourceMock(BitbucketRepositoryType.GIT);
-        String remote = source.getRemote("amuniz", "test", source.getRepositoryType());
+        String remote = source.getRemote(repoOwner, repoName, source.getRepositoryType());
```

Comment:
Well, nothing special... I probably forgot to replace them during a test refactor. I'm ok with the change.